src/main/features/claim/views/task-list.njk

```diff
@@ -6,23 +6,23 @@
   <div class="grid-row">
     <div class="column-two-thirds">
       <div class="task-group">
-        {{ taskListHeading(beforeYouStart.position, t(beforeYouStart.name)) }}
+        {{ taskListHeading(beforeYouStart.name) }}
```

Comment:
Is this still able to be translated?

Reply:
translataion is moved to `taskListHeading`
https://github.com/hmcts/cmc-common-frontend/pull/106